src/components/Differential/PlotsOverlay.jsx

```diff
@@ -160,7 +162,27 @@ class PlotsOverlay extends PureComponent {
       //   //  </div>
       // );
 
-      if (this.props.differentialPlotTypes && this.props.plotOverlayData) {
+      if (differentialPlotTypes && plotOverlayData) {
+        let differentialPlotDescription = null;
+        let currentDifferentialPlotDescriptions =
+          differentialPlotDescriptions?.[differentialModel] || {};
+        const currentPlotText =
+          options?.[activeTabIndexPlotsOverlayVar]?.text || null;
+        if (!isObjectEmpty(currentDifferentialPlotDescriptions)) {
```

Comment:
The JSON from the backend is a bit atypical, so I made this work for plots.  If we could get the data in with other plots or tests data then we wouldn't need this, but with reviewer response deadline Wednesday it has to suffice